jaxrs-api/src/main/java/jakarta/ws/rs/core/Cookie.java

```diff
@@ -216,4 +234,104 @@ public boolean equals(final Object obj) {
         }
         return true;
     }
+
+    /**
+     * JAX-RS {@link Cookie} builder class.
```

Comment:
Despite simply repeating the obvious, it might be beneficial to have some idea here what the builder actually is good for. There was a good reason why you invested your time to write it, so let the readers learn from this JavaDocs. :-)

Reply:
Well, builder pattern is a well known pattern now so I didn't think it worth explaining people why this pattern is better than unfriendly telescoping constructor (anti-)pattern.
By the way I got inspired by `OutboundSseEvent.Builder` javadoc so I thought it is acceptable.

Anyway I will changed it into something more like `jakarta.ws.rs.sse.SseEventSource.Builder`. :+1: